File: stress/stress.c
#include <wchar.h>
#include <stdint.h>
#include <string.h>
#include <stddef.h>
#include <math.h>

#include "common.h"

void stress(
	int i, unsigned int u,
	double d, long double ld,
	char *s, wchar_t *ws,
	signed char hhi, unsigned char hhu, short hi, unsigned short hu,
	long li, unsigned long lu, long long lli, unsigned long long llu,
	intmax_t ji, uintmax_t ju, size_t zi, ssize_t zu, ptrdiff_t ti, uintptr_t tu
) {
#include "stress.h"
}

int main() {
	#include "tests.h"
}

File: stress/stress.js
#ifndef JAVASCRIPT
#define JAVASCRIPT
#endif
var sprintf = require("../");
var printf = function() {
	var o = sprintf.sprintf.apply(null, arguments);
	process.stdout.write(o);
}

function stress(
	i, u,
	d, ld,
	s, ws,
	hhi, hhu, hi, hu,
	li, lu, lli, llu,
	ji, ju, zi, zu, ti, tu
) {
#include "stress.h"
}

#include "common.h"

#include "tests.h"
